examples/graphql/server-express.js

```diff
@@ -5,10 +5,33 @@ require('./tracer');
 const express = require('express');
 const { graphqlHTTP } = require('express-graphql');
 const buildSchema = require('./schema');
+const otel = require('@opentelemetry/api');
 
 const schema = buildSchema();
 
+const testMw = function (req, res, next) {
+  otel.context.with(
+    otel.propagation.setBaggage(
+      otel.context.active(),
+      otel.propagation.createBaggage({
+        'test': { value: 'testValue' },
+      })
+    ),
+    () => {
+      console.log('attached baggage', otel.propagation.getBaggage(otel.context.active()));
+      next();
```

Comment:
Express break context propagation (see https://github.com/open-telemetry/opentelemetry-js-contrib/blob/main/plugins/node/opentelemetry-instrumentation-express/src/instrumentation.ts#L263) so you'll need to use `context.bind` when calling next here